### PR TITLE
Port constraint_sampler_manager_loader to ROS2

### PR DESCRIPTION
**Pull request by anasarrak**

### CODE REVIEW

**Comment by vmayoral** [comment]
CI failed @anasarrak, can you review please? @ahcorde can you help?

File: moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp
Comment (LanderU) [inline moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp]: This caused the error on the CI:

```bash
/root/moveit2/moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp:93:1: error: ‘node_’ does not name a type; did you mean ‘mode_t’?
 node_->get_logger(),
```

**Review by vmayoral — APPROVED** [review]
Let’s wait for the CI before merging but LTGM from my side.

**Comment by ahcorde** [comment]
Linux is passing but OSX is failing

**Comment by anasarrak** [comment]
OSX is failing because of [this](https://github.com/AcutronicRobotics/geometric_shapes/blob/e02b97df2fa02dedf2f45cd10f3fe47cfd546b63/CMakeLists.txt#L79) line on geoetric_shapes

**Review by ahcorde — APPROVED** [review]

**Pull request merged**
